tool_collections/bamtools/bamtools_split/bamtools-split.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="bamSplit" name="Split" version="0.0.2" force_history_refresh="True">
+<tool id="bamSplit" name="Split" version="0.0.3" force_history_refresh="True">

```

Comment:
You should probably remove ` force_history_refresh="True"` (the XSD linter is complaining about it).